Geometry/TrackerNumberingBuilder/interface/GeometricDet.h

```diff
@@ -4,6 +4,7 @@
 #include "CondFormats/GeometryObjects/interface/PGeometricDet.h"
 #include "DetectorDescription/Core/interface/DDExpandedView.h"
 #include "DetectorDescription/Core/interface/DDSolidShapes.h"
```

Comment:
`Core` headers, see my comment above

Reply:
Andres is right. There are still problems here. I think line 6 needs to be deleted. I don't see the old `DDSolidShape` being used here.
Also, migration code has to provided for the old `DDExpandedView`.

Reply:
Ha  - I missed this. L6 can indeed go. As for `DDExpandedView` - what it provides for this file is two types that are effectively vectors of ints, so I've moved those defs into this file. However, I also see ` using GeoHistory = std::vector<DDExpandedNode>;`, even though `GeoHistory` is not used in this file. Perhaps that can be removed - then `DDExpandedView` is not needed in this file at all.

I assume there is a good reason there's no direct dd4hep copy of `DDExpandedView` / `DDExpandedNode` (yet) - but if central changes are needed there they could then happen outside of this PR. In any case I don't think I'm the right person to make those changes (I definitely don't know enough about the requirements to be able to do this correctly)